### ISSUE TITLE
[Issue/Own mistake] AND-Operator issues

### ISSUE THREAD

**Issue by accessdenied0** [CyclopsMC/IntegratedDynamics#573]
#### Issue type:

- :bug: Bug
Operator AND seems not to work
____

#### Short description:

My goal was to connect a Clock with a Entity-Detector to emit a Clock-Impulse everytime a Entity (Item) was in front of it. I converted the _Entity-List_ to a Is _notEmpty_ and stored it with the Clock variable in the Variable container. I used AND with Variable _(IsNotEmpty)_ and _(Clock)_. Normally if _IsNotEmpty_ would be true, the pulse of the _CLOCK_ would be go through. But it always returns _false_. I now tried every variable each, every variable puts the right output. I used _(IsNotEmpty)_ AND _(IsNotEmpty)_ as a test, because this should be true in any case. AND returns false.
Also I noticed that there is no xor as binary and nxor doesn't exist at all.

#### Steps to reproduce the problem:

[Picture of the setup](http://puu.sh/AmSIi/85f854bf32.png)

#### Expected behaviour:
____

#### Versions:

- This mod: IntegratedDynamics-1.12.2-0.11.11
- Minecraft: 1.12.2
- Forge: 14.23.3.2655

**Comment by rubensworks** [comment]
Just tested this myself, and everything seems to work just fine here. No issues at all.

Could you try reproducing this without any other mods installed? And if you can still reproduce it, please give me a detailed step-by-step guide of what you did.

**Comment by accessdenied0** [comment]
Strangely enough: After I tear it all down and [made a different setup](http://puu.sh/ApXiH/21e91bc47b.jpg), it worked. I'm really not sure because I did everything the same, except I used a new set of Values. I didn't changed anything else...

To another point: Is it possible to add a XOR (and maybe a NXOR) modifier for Boolean? That would be really convenient.

I'm really sorry If I caused trouble here. I'm glad it works now...
